tss-esapi/tests/Dockerfile-ubuntu

```diff
@@ -18,10 +18,12 @@ RUN cd tpm2-tss \
 	&& make install \
 	&& ldconfig
 
+FROM tpm2-abrmd 
+
 FROM tpm2-tss AS tpm2-tools
 # Download and install TPM2 tools
-RUN git clone https://github.com/tpm2-software/tpm2-tools.git --branch 4.1
+RUN git clone https://github.com/tpm2-software/tpm2-tools.git --branch 5.6
 RUN cd tpm2-tools \
 	&& ./bootstrap \
-	&& ./configure --enable-unit \
```

Comment:
Why is `--enable-unit` gone? Removed in a new version?

Reply:
My question is why was it there in the first place? We do not need to run the unit tests in tpm2-tss so why should we build them?

And in later versions of tpm2-tss it actually becomes an issue because the tests requires that tpm2-abrmd is installed in order to be able to build.